TFNetworkLayer.py

```diff
@@ -1892,14 +1893,18 @@ def __init__(self, activation, with_bias=True, grad_filter=None,
     :param str forward_weights_init: see :func:`TFUtil.get_initializer`
     :param str recurrent_weights_init: see :func:`TFUtil.get_initializer`
     :param str|float bias_init: see :func:`TFUtil.get_initializer`
+    :param int|None out_feature_dim_axis: if set, will convert feature dim of the input and output data
```

Comment:
This option is not really needed. The option `out_type` already supports to define a custom `feature_dim_axis`.
So please remove it.